### PR TITLE
[HOC][PoemBot] Add poem selector dropdown

### PR DESCRIPTION
**Pull request by ajpal**
Adds a dropdown to select a poem:
Aug-12-2021 14-34-50

No change in gamelab or non-poem-bot-spritelab:
image
Screen Shot 2021-08-12 at 2 35 45 PM

Unit tests to come in a follow-up PR

### CODE REVIEW

File: apps/src/p5lab/spritelab/PoemSelector.jsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {setPoem} from '../redux/poemBot';
+import msg from '@cdo/spritelab/locale';
+import {APP_WIDTH} from '../constants';
+
+const poems = [
```
Comment (ajpal) [inline apps/src/p5lab/spritelab/PoemSelector.jsx:8]: These aren't the final poems. Those will come from Amy later.

Comment (ajpal) [reply apps/src/p5lab/spritelab/PoemSelector.jsx:8]: (and will need to be translated)

**Review by epeach — APPROVED** [review]
LGTM

**Comment by ajpal** [comment]
Closed in favor of https://github.com/code-dot-org/code-dot-org/pull/42718

**Pull request closed without merging**